src/assets/scripts/client/aircraft/StripView/StripViewModel.js

```diff
@@ -381,6 +420,8 @@ export default class StripViewModel extends BaseModel {
         this._flightPlan = flightPlan;
         this._categoryClassName = this._buildClassnameForFlightCategory(aircraftModel);
         this.isDeparture = aircraftModel.isDeparture();
+        // TODO: remarks
```

Comment:
I'm a huge fan of `TODO` comments, as I'm sure you've seen already.  If you're planning to address this with the PR, please do so.  Otherwise, could you add a little more information here about what exactly this `TODO` is for?

Reply:
I hate `TODO`. Always feels like an excuse to be lazy and not do what needs to be done.

Anyways, I have no idea what the remarks field is intended for. I found it in the template and added the missing code. 

@erikquinn  any thoughts?

Reply:
It's the remarks on the flight plan. Whenever somebody files a flight plan, they can add notes to it, and probably the reason it was built in is because that's where they would go.

Reply:
@erikquinn  Do we support remarks in flight plans already?

Reply:
no, it would be an easy add, but I don't see the need. It was just easy enough to set up a bunch of fields when building the flight strips so they'd be plug-and-play with future things we could add to them.

Reply:
okay, so is there anything you want me to do here or do we keep it as it is right now?

Reply:
no, you can ignore anything regarding remarks. I saw a `// TODO: remarks` somewhere you could also remove.

Reply:
done